packages/cli/src/commands/index.ts

```diff
@@ -39,3 +39,10 @@ export default new Map([
   ['teams', 'teams'],
   ['whoami', 'whoami'],
 ]);
+
+if (process.env.CUSTOM_ENVIRONMENTS) {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** add comment

Can we add a comment here explaining why this is here? This is temporary, correct?